mne/viz/tests/test_decoding.py

```diff
@@ -21,15 +22,10 @@
 
 warnings.simplefilter('always')  # enable b/c these tests throw warnings
 
-# Set our plotters to test mode
-
-tmin, tmax = -0.2, 0.5
-event_id = dict(aud_l=1, vis_l=3)
-event_id_gen = dict(aud_l=2, vis_l=4)
-
 
 @requires_sklearn
```

Comment:
...it is slightly less DRY, but it wins for clearness of code. It makes it clearer which tests have additional package requirements.